examples/guided.py

```diff
@@ -339,7 +231,14 @@ def perform_installation(mountpoint):
 	Only requirement is that the block devices are
 	formatted and setup prior to entering this function.
 	"""
+
+
 	with archinstall.Installer(mountpoint, kernels=archinstall.arguments.get('kernels', 'linux')) as installation:
+		# Mount all the drives to the desired mountpoint
+		# This *can* be done outside of the installation, but the installer can deal with it.
+		for drive in archinstall.arguments['harddrives']:
+			installation.mount_ordered_layout(archinstall.storage['disk_layouts'][drive])
```

Comment:
There might be mount points that overlap between drives, causting them to be mounted in the incorrect order, `mount_ordered_layout()` should take all drives into account.